templates/tease-author.twig

```diff
@@ -8,7 +8,7 @@
 				{{ post.get_author_override }}
 
 				{% for page_type in post.get_custom_terms %}
-					<a href="/?s=&orderby=relevant&f[ptype][{{ page_type.name }}]={{ page_type.term_id }}" class="search-result-item-head no-btn tag-item tag-item--main page-type">{{ page_type.name|e('wp_kses_post')|raw }}</a>
+					<a href="/page_type/{{ page_type.slug }}" class="search-result-item-head no-btn tag-item tag-item--main page-type">{{ page_type.name|e('wp_kses_post')|raw }}</a>
```

Comment:
@pablocubico please make these links either relative permalinks (for example using wp_make_link_relative) or use the full link for the term (in production we have base urls like greenpeace.org/international/ ).